### PR TITLE
Fix #11: Add emacs keybindings and use commands.

### LINKED ISSUE #11: Add emacs keybinds for up/down when selecting file
Proposed add for Emacs ctrl-n and ctrl-p for selecting next/prev file in list. 
Also, ctrl-i/ctrl-k are popular movement key combos.

### PR DESCRIPTION
**Pull request by Osmose**
Add extra keybindings for navigation through the file list, and switch to using
commands for implementing keybindings so users can customize them to their
heart's content.

@nomatics Would you be willing to pull this down and test out if it works? Also if you want to glance through the code and leave any feedback that'd be great! :D

@mythmon r? if you're tired of fixing your media center :P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,44 @@ symbol will add it as a project directory.
...
+  <dt>`advanced-open-file:move-cursor-up`</dt>
+  <dd>Move the cursor/highlight for the currently selected file up.</dd>
+
+  <dt>`advanced-open-file:move-cursor-down`</dt>
+  <dd>Move the cursor/highlight for the currently selected file down.</dd>
+</dl>
+
+The following extra keybindings are included by default:
+
+Action                                | Extra Keys
```
Comment (mythmon) [inline README.md]: Calling these "extra keybindings" and "Extra keys" seems confusing to me. What makes them extra?

Comment (Osmose) [inline README.md]: How about "Alternate keybindings"?

Comment (mythmon) [inline README.md]: I guess I'm confused; what are these an alternative to? The arrow keys?

**Comment by mythmon** [comment]
I got this working in dev mode locally, and turned on the key binding resolver debugger (`ctrl+.` for me). It does what I expect it to, and the code is a lot prettier. My confusion in the README is minor, and doesn't need to be addressed if you don't have any clever ideas. r+

File: README.md (hunk 1 above)
Comment (Osmose) [inline README.md]: Yeah. Earlier in the README it explains what you can do and what keybindings are available; this section follows, describing alternative keybindings that are included by default for people who are more used to these common alternatives.

Comment (mythmon) [inline README.md]: Ok. My problem was just that I didn't read the whole readme. Sounds good like this then.

**Comment by Osmose** [comment]
Thanks for the review!

**Pull request merged**